src/styles/index.scss

```diff
@@ -1,21 +1,29 @@
 @import 'variables';
 
 .mirador {
-  &-app {
-    height: 100%;
-    position: relative;
-    width: 100%;
+  &-viewer {
+    position: absolute;
```

Comment:
I'm seeing some style warnings in our `npm run lint` task. Should we fix those now or spin off a ticket for that?